tests/vm_generator.go

```diff
@@ -0,0 +1,95 @@
+package tests
+
+import (
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func randName(name string) string {
+	return name + "-" + rand.String(5)
+}
+
+func getBaseVMISpec() *v1.VirtualMachineInstanceSpec {
+	return &v1.VirtualMachineInstanceSpec{
+		Domain: v1.DomainSpec{
+			Resources: v1.ResourceRequirements{
+				Requests: k8sv1.ResourceList{
+					k8sv1.ResourceMemory: resource.MustParse("64M"),
```

Comment:
Please use 128Mi
64Mi is not enough
see please https://github.com/kubevirt/kubevirt/pull/5039

Reply:
done